tron/serialize/runstate/dynamodb_state_store.py

```diff
@@ -58,28 +63,46 @@ def alert(self, msg: str):
         }
         pysensu_yelp.send_event(**result_dict)
 
-    def __getitem__(self, key: ShelveKey) -> object:
-        """
-        It returns an object which is deserialized from binary
-        """
+    def _get_items(self, keys: list) -> object:
+        items = []
         table_name = self.name.replace('/', '-')
-        keys = [{'key': {'S': str(key)}, 'index': {'N': str(index)}} for index in range(self._get_num_of_partitions(key))]
-        if not keys:
-            return None
-        vals = self.client.batch_get_item(
-            RequestItems={
-                table_name: {
-                    'Keys': keys,
-                    'ConsistentRead': True
-                },
-            }
-        )['Responses'][table_name]
-        vals.sort(key=lambda x: x['index']['N'])
-        res = bytearray()
-        for val in vals:
-            res += bytes(val['val']['B'])
+        for i in range(0, len(keys), 100):
+            vals = self.client.batch_get_item(
+                RequestItems={
+                    table_name: {
+                        'Keys': keys[i:min(len(keys), i + 100)],
+                        'ConsistentRead': True
+                    },
+                }
+            )['Responses'][table_name]
+            items.extend(vals)
+        return items
+
+    def _get_first_items(self, keys: list):
```

Comment:
How about renaming these _get_first_partitions, _get_remaining_partitions, and _merge_partitions?